src/libs/fetcher.ts

```diff
@@ -1,10 +1,42 @@
+// Need a package that can dynamically switch between the JS `fetch`
+// and the `node-fetch` based on usage.
+import fetch from "isomorphic-unfetch";
+import { IncomingMessage } from "http";
```

Comment:
Please use the types for requests from next instead of http:
https://nextjs.org/docs/basic-features/typescript
```typescript
import { NextApiRequest, NextApiResponse } from 'next'
```

Reply:
Done